pkg/oc/lib/newapp/app/templatelookup.go

```diff
@@ -13,14 +13,14 @@ import (
 	kapi "k8s.io/kubernetes/pkg/apis/core"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/resource"
 
+	templatev1 "github.com/openshift/api/template/v1"
+	templatev1client "github.com/openshift/client-go/template/clientset/versioned/typed/template/v1"
 	"github.com/openshift/origin/pkg/oc/util/ocscheme"
```

Comment:
While you're at it, figure out what this is used for and if this can be replaced with kubectl's scheme.